### ISSUE TITLE
Unable to connect to HA through app

### ISSUE THREAD

**Issue by derkork** [home-assistant/android#1578]
**Home Assistant Android version:**
2021.5.1-full

**Android version:**
7.1.2

**Phone model:**
Google Nexus 7 (2013)

**Home Assistant version:**
 2021.6.0

**Last working Home Assistant release (if known):**
N/A

**Description of problem:**
I can not connect to HA. My server is detected but when I click on its name, loading starts, then the screen remains frozen on "initializing", and I can not enter my login credentials. Using Brave Browser, on the same tablet I can connect to the server and get a proper login screen. I connect locally with HTTP so it cannot be a certificate problem.
 
**Traceback (if applicable, to get the logs you may refer to: https://companion.home-assistant.io/docs/troubleshooting/faqs/#android-crash-logs):**

I cannot currently access the logs because I cannot log into the application.

**Screenshot of problem:**

![20210604_000208](https://user-images.githubusercontent.com/327257/120717793-6f047b00-c4c8-11eb-9707-50003f99e7e7.jpg)

**Additional information:**
Android Web View is updated to the latest version. The app works fine on my newer Galaxy S10+ using the same settings, so I suspect it is some peculiarity with the older Android version of my Tablet.

**Comment by dshokouhi** [comment]
you are on a custom ROM you need to switch the default webview to the correct implementation...how you do it on your custom ROM depends on the ROM.  Android System Webview needs to be a system installed app and not just installed.

Duplicate of #395

**Comment by derkork** [comment]
Thank you very much! I changed the web view implementation in the settings to Google WebView and now I see a login screen.

**Comment by derkork** [comment]
@dshokouhi  I just found your commit where you added this to the troubleshooting information (https://github.com/home-assistant/companion.home-assistant/pull/168/commits/81d9302c06e4c12da2c45bcd2cd279ff0fe12f1d) but I cannot see this information at (https://companion.home-assistant.io/docs/troubleshooting/faqs/). Did I look at the wrong place?

**Comment by daanbreur** [comment]
@derkork No the PR for it is closed and not merged. They dont want to support custom roms. https://github.com/home-assistant/companion.home-assistant/pull/168

**Comment by Blasphemes** [comment]
Hi Everyone,

I was getting the "Forbidden" error when accessing my Home Assistant server (remotely, and on a different IP address) via the Android App. I then also tried accessing the server from my Android web browser and that threw the same "Forbidden" error. I solved the issue when I found an ip_bans.yaml file in my server's working directory. The file contained the IP address of my phone.

I inspected my configuration.yaml file and found two settings:
```
  ip_ban_enabled: True
  login_attempts_threshold: 10
```
So I deleted the ip_bans.yaml file, reset the login_attempts_threshold to 50 (from 10), restarted the server, and was able to configure the app and connect. Perhaps others are having this problem, so I hope this information is helpful to some.

Michael
